hr_cae_contract/models/hr_contract.py

```diff
@@ -50,15 +50,28 @@ def get_latest(self):
             limit=1,
         )
 
+    def has_termination_contract(self):
+        return bool(
+            self.contract_ids.search(
+                [
+                    ("contract_group_id", "=", self.id),
+                    ("echelon", "=", "termination"),
+                ]
+            )
+        )
+
 
 class ContractType(models.Model):
 
     _inherit = "hr.contract.type"
 
     echelon = fields.Selection(
-        [("main", "Main"), ("amendment", "Amendment")],
+        [
+            ("main", "Main"),
+            ("amendment", "Amendment"),
+            ("termination", "Termination"),
+        ],
         string="Echelon",
-        default="main",
```

Comment:
Because otherwise it sets this as default for the echelon of the three existing contract types in Odoo HR, and these three also start showing up when making a new main type contract.

Reply:
Another option might be to launch a pre-hook and give all existing types a echelon e.g. `other`, but that seems a bit overkill?
The default was not really useful anyway.

Reply:
good reasoning, ok to remove the default value 👍